src/common/Network/PacketDef/Zone/ServerZoneDef.h

```diff
@@ -283,6 +283,105 @@ namespace Sapphire::Network::Packets::Server
     char unk2[0x4]; // This has probs something to do with the support desk (inquiry id?)
   };
 
+  struct FFFXIVIpcMarketBoardItemListingCount : FFXIVIpcBasePacket< MarketBoardItemListingCount >
+  {
+    uint32_t itemCatalogId;
+    uint32_t unknown1; // does some shit if nonzero
+    uint16_t requestId;
+    uint16_t quantity; // high/low u8s read separately?
+    uint32_t unknown3;
+  };
+
+  struct FFXIVIpcMarketBoardItemListing : FFXIVIpcBasePacket< MarketBoardItemListing >
+  {
+    struct ItemListing // 152 bytes each
+    {
+      uint32_t unknown; // Changes if multiple packets are sent for a single item
+      uint8_t padding;
+      uint16_t unknown1;
+      uint8_t padding1;
+      struct unknown2
+      {
+        uint32_t unknown3;
+        uint16_t unknown4;
+        uint16_t unknown5;
+      } unknown6[3];
+      uint32_t pricePerUnit;
+      uint32_t unknown7;
+      uint32_t itemQuantity;
+      uint32_t itemId;
+      uint16_t unknown8;
+      uint16_t unknown9;
+      uint32_t unknown10;
+      uint16_t unknown11; // Always 30000
+      uint16_t padding3;
+      uint16_t materiaValue[5];
+      /**
+       * auto materiaId = (i & 0xFF0) >> 4;
+       * auto index = i & 0xF;
+       * auto leftover = i >> 8;
+       */
+      uint32_t padding4;
+      char retainerName[64];
+      bool hq;
+      uint8_t materiaCount;
+      uint8_t padding5;
+      uint8_t retainerCity;
+      /**
+       * 0x01 Limsa Lominsa
+       * 0x02 Gridania
+       * 0x03 Ul'dah
+       * 0x04 Ishgard
+       * 0x07 Kugane
+       * 0x0A Crystarium
+       */
```

Comment:
this should be above retainerCity, not below it

Reply:
got it, i'll move the other one too